src/cpu/core_dynrec/risc_ppc64le.h

```diff
@@ -564,10 +564,11 @@ static void inline gen_load_param_imm(Bitu imm, Bitu param)
 }
 
 // load an address as param'th function parameter
-// 32-bit
+// can be 64-bit, especially in PIE code
 static void inline gen_load_param_addr(Bitu addr, Bitu param)
 {
-	gen_load_param_imm(addr, param);
+	// "gen_mov_qword_to_reg"
+	gen_mov_qword_to_reg_imm(RegParams[param],(uint64_t)addr);
```

Comment:
Very minor:  Do we need this comment?  It's basically just the name of the function.

Reply:
It's just a mental note to myself since this was converted from x86_64 and the function names aren't quite the same. I may harmonize this later to get rid of the comments.